csharp/test/IceSSL/configuration/AllTests.cs

```diff
@@ -45,11 +45,13 @@ private static Dictionary<string, string>
     createServerProps(Ice.Properties defaultProperties)
     {
         Dictionary<string, string> result = new Dictionary<string, string>();
-        result["IceSSL.DefaultDir"] = defaultProperties.getProperty("IceSSL.DefaultDir");
-        result["Ice.Default.Host"] = defaultProperties.getProperty("Ice.Default.Host");
-        if (defaultProperties.getProperty("Ice.IPv6").Length > 0)
+        result["IceSSL.DefaultDir"] = defaultProperties.getIceProperty("IceSSL.DefaultDir");
```

Comment:
I find this conversion logic unexpected - and ideally we should fix it and just "convert" with:
https://github.com/zeroc-ice/ice/blob/2846e40a8ea07d870d762d89b2075071ab04aba2/csharp/src/Ice/Properties.cs#L285

This is however not the focus of this PR.